src/ptvsd/adapter/messages.py

```diff
@@ -342,6 +357,9 @@ def continue_request(self, request):
     @_only_allowed_while("configuring", "running")
     def ptvsd_systemInfo_request(self, request):
         result = {"ptvsd": {"version": ptvsd.__version__}}
+        if self._no_debug:
+            return result
+
```

Comment:
This shouldn't be possible, since nodebug will never enter any of the states in which this request is allowed - so no need to check.